.github/workflows/run_benchmarks.yml

```diff
@@ -45,21 +45,21 @@ jobs:
           cd benchmarks
           python run_benchmarks.py --verbose --pyccel-config-files pyccel_gnu.json pyccel_intel.json --pythran-config-files pythran_gnu.config pythran_intel.config
           cd ..
-          FILE=version_specific_results/devel_performance_310.md
+          FILE=version_specific_results/devel_performance_311.md
           echo "### Performance Comparison (as of $(date))" > ${FILE}
           cat benchmarks/bench.out >> ${FILE}
           python analysis/plot_results_figures.py ${FILE}
         shell: bash
         working-directory: ./.
       - name: Record Failure
-        run: echo "Devel branch benchmarks failed on python 3.11!" > version_specific_results/devel_performance_310.md
+        run: echo "Devel branch benchmarks failed on python 3.11!" > version_specific_results/devel_performance_311.md
         if: ${{ failure() }}
       - name: Add & Commit
         uses: EndBug/add-and-commit@v9
         if: ${{ always() }}
         with:
           message: 'Update performance comparison'
-          add: "['version_specific_results/devel_performance_310.md', 'version_specific_results/devel_performance_310_compilation.svg', 'version_specific_results/devel_performance_310_execution.svg', 'version_specific_results/devel_performance_310_requirements.txt']"
+          add: "['version_specific_results/devel_performance_311.md', 'version_specific_results/devel_performance_311_compilation.svg', 'version_specific_results/devel_performance_311_execution.svg', 'version_specific_results/devel_performance_311_requirements.txt']"
```

Comment:
If you modify the name of the generated file I think you probably also need to update .github/actions/create_readme

Reply:
That file looks general enough to me: https://github.com/pyccel/pyccel-benchmarks/blob/main/.github/actions/create_readme/action.yml

Reply:
True! Should we use VERSION there too?

Reply:
No. `VERSION` refers to the version of Pyccel. `310` refers to the version of Python.

This line is equivalent to the line above where you have changed `devel_performance_310` to `devel_performance_311`

Reply:
Done in [fe24332](https://github.com/pyccel/pyccel-benchmarks/pull/40/commits/fe243321cb64eae4f2afe5e04e009f32510d15f8)